userdocs/profiles.dev/src/components/Schema.js

```diff
@@ -0,0 +1,282 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Modifications:
+  Copyright 2021 Weaveworks
+*/
+
+import React, { useState, useEffect } from "react";
+
+const Schema = (input) => {
```

Comment:
```suggestion
const Schema = ({jsonFile}) => {
```
you can use the prop directly